Add staking and operators guide (#199)
subspace/subspace-docs: versioned_docs/version-latest/operators_and_staking/intro.md

- **Higher [Hardware Requirements](#operator-hardware-requirements)**: Operators require more substantial hardware capabilities, as they must execute complex transactions.
- **Require Initial Investment:** Operators are required to stake a certain amount of SSC. If an operator acts maliciously, their stake is at risk of being **slashed**. Engaging in such malicious behavior carries significant penalties, providing crypto-economic security to execution.
- **Pre-Validation and Batching**: Operators pre-validate and batch transactions into bundles through a stake-weighted election process.
- **Deterministic Execution**: The operators execute transactions in a specific, deterministic order, producing state commitments in the form of execution receipts.
- **Secondary Network Role**: Monitors the Domain chain for malicious activity and submits fraud proofs to consensus chain.
- **Supports Various Environments**: Can support different smart contract execution environments like the Ethereum Virtual Machine (EVM) or Web-Assembly (WASM).

### Operator hardware requirements

:::note

> The hardware requirements have not been benchmarked, and these are our best estimates. We're open to your feedback if you feel that the requirements listed here are too high or too low. 

Reviewer: ```suggestion
The hardware requirements have not been benchmarked, and these are our best estimates. We would apptreciate your feedback if you feel that the requirements listed here are too high or too low. 
```

Reviewer: Nice! You fixed the typo I introduced too! :)
